### PR TITLE
Add README.md for elasticdl_client folder.

### PR DESCRIPTION
**Pull request by brightcoder01**

### CODE REVIEW

File: elasticdl_client/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ElasticDL Client
+
+This is the client command-line tool for ElasticDL. We can use it to build
```
Comment (skydoorkai) [inline elasticdl_client/README.md]: This is more straightforward:

`ElasticDL client is a command-line tool. Users can use it to prepare and submit ElasticDL jobs.`

Comment (brightcoder01) [reply elasticdl_client/README.md]: Good suggestion. Make some update based on this comment.

File: elasticdl_client/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# ElasticDL Client
+
+ElasticDL client is a command-line tool. Users can use it to prepare
+Docker images for a distributed ElasticDL job and submit a train, evaluate
```
Comment (workingloong) [inline elasticdl_client/README.md]: "Docker images" -> "the Docker image" 
Because a job only needs an image.

Comment (brightcoder01) [reply elasticdl_client/README.md]: Good catch, updated.

**Review by wangkuiyi — APPROVED** [review]

**Pull request merged**